packages/cli/src/types.ts

```diff
@@ -208,22 +208,21 @@ export enum ProjectEnvTarget {
 export enum ProjectEnvType {
   Plaintext = 'plain',
   Secret = 'secret',
+  Encrypted = 'encrypted',
   System = 'system',
 }
 
 export interface ProjectEnvVariable {
+  id: string;
```

Comment:
I confirmed with @lucleray that all env vars now have an ID thanks to a previous migration